### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by maikksmt**
Translated into German.

- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by jens1o** [comment]
Could you rename it back and I'll merge it? Reason in https://github.com/jens1o/de-opensource.guide/pull/4

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,72 +2,72 @@
 layout: default
 ---
 
-# Contributing to Open Source Guides
+# Zu Open Source Leitfäden beitragen
 
-Thanks for checking out the Open Source Guides! We're excited to hear and learn from you. Your experiences will benefit others who read and use these guides.
+Danke, dass Sie sich die Open Source Leitfäden ansehen! Wir freuen uns, von Ihnen zu hören und zu lernen. Ihre Erfahrungen kommen anderen zugute, die diese Leitfäden lesen und verwenden.
```
Comment (jens1o) [inline CONTRIBUTING.md]: `du`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,72 +2,72 @@
 layout: default
 ---
 
-# Contributing to Open Source Guides
+# Zu Open Source Leitfäden beitragen
```
Comment (jens1o) [inline CONTRIBUTING.md]: `Open-Source` yet again

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,72 +2,72 @@
...
+## Die Art von Beiträgen, die wir suchen
+In erster Linie ist dieses Projekt ein Forum, um Open Source-Best Practices zu diskutieren und es dann in einem Leitfaden zu dokumentieren, wenn wir einen Konsens gefunden haben. Ihr erster Beitrag könnte eine neue Konversation sein oder eine vorhandene Konversation zu Best Practices ergänzen. Sie können dies unter [Issues](https://github.com/github/opensource.guide/issues) tun.
+Es gibt auch viele Möglichkeiten, wie Sie direkt zu den Leitfäden beitragen können (in absteigender Reihenfolge der Bedürfnisse):
 
-There are also many ways you can directly contribute to the guides (in descending order of need):
+* Behebung von redaktionellen Inkonsistenzen und Ungenauigkeiten.
+* Fügen Sie Geschichten, Beispiele und Anekdoten hinzu, die einen Punkt veranschaulichen.
+* Überarbeitung der Formulierungen, um zugänglicher und freundlicher zu sein.
+* [Leitfäden in andere Sprachen übersetzen](docs/uebersetzungen.md)
```
Comment (jens1o) [inline CONTRIBUTING.md]: I'm not sure, I think the links should stay intact, as long as we didn't translated it.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,72 +2,72 @@
...
 
-Before we get started, here are a few things we expect from you (and that you should expect from others):
+* Sei nett und nachdenklich in deinen Gesprächen rund um dieses Projekt. Wir alle kommen aus unterschiedlichen Hintergründen und Projekten, was bedeutet, dass wir wahrscheinlich unterschiedliche Perspektiven haben, wie "Open Source" gemacht wird. Versuche anderen zuzuhören, anstatt sie davon zu überzeugen, dass dein Weg korrekt ist.
+* Open Source Leitfäden werden mit einem [Verhaltenskodex für Mitwirkende] (./VERHALTENSKODEX.md) veröffentlicht. Durch die Teilnahme an diesem Projekt stimmen Sie zu, die Bedingungen einzuhalten.
+* Wenn Sie eine Pull-Anfrage öffnen, stellen Sie bitte sicher, dass Ihr Beitrag alle Tests besteht. Wenn Testfehler auftreten, müssen Sie diese beheben, bevor wir Ihren Beitrag zusammenführen können.
+* Wenn Sie Inhalte hinzufügen, überlegen Sie bitte, ob sie von großem Wert sind. Bitte fügen Sie keine Verweise oder Links zu Dingen hinzu, die Sie oder Ihr Arbeitgeber erstellt haben. Andere werden dies tun, wenn sie die Inhalte zu schätzen wissen.
```
Comment (jens1o) [inline CONTRIBUTING.md]: `du`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,72 +2,72 @@
...
 
-* Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on "how open source is done." Try to listen to others rather than convince them that your way is correct.
-* Open Source Guides are released with a [Contributor Code of Conduct](./CODE_OF_CONDUCT.md). By participating in this project, you agree to abide by its terms.
-* If you open a pull request, please ensure that your contribution passes all tests. If there are test failures, you will need to address them before we can merge your contribution.
-* When adding content, please consider if it is widely valuable. Please don't add references or links to things you or your employer have created as others will do so if they appreciate it.
+## Wie man beiträgt
 
-## How to contribute
+Wenn Sie etwas beitragen möchten, suchen Sie zunächst in den [Issues] (https://github.com/github/opensource.guide/issues) und [Pull Requests] (https://github.com/github/opensource). .guide / pulls) um zu sehen, ob jemand anderes eine ähnliche Idee oder Frage gestellt hat.
```
Comment (jens1o) [inline CONTRIBUTING.md]: `du`

And please remove the spaces and change the links, so they pass them on to this repo.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -2,72 +2,72 @@
...
 
-If you'd like to contribute, start by searching through the [issues](https://github.com/github/opensource.guide/issues) and [pull requests](https://github.com/github/opensource.guide/pulls) to see whether someone else has raised a similar idea or question.
+Wenn Ihre Idee nicht aufgeführt ist und Sie glauben, dass sie in die Ziele dieses Leitfadens passt, führen Sie einen der folgenden Schritte aus:
+* ** Wenn Ihr Beitrag geringfügig ist, ** wie z. B. ein Tippfehler, öffnen Sie eine Pull-Anfrage.
```
Comment (jens1o) [inline CONTRIBUTING.md]: Remove the spaces after and before the astericks

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -2,72 +2,72 @@
...
 
-## Style guide
-If you're writing content, see the [style guide](./docs/styleguide.md) to help your prose match the rest of the Guides.
+Wenn Sie Inhalte schreiben, lesen Sie die [Gestaltungsrichtlinie] (./docs/gestaltungsrichtlinie.md), um Ihre Prosa mit den restlichen Leitfäden abzustimmen.
 
-## Setting up your environment
+## Einrichten Ihrer Umgebung
 
-This site is powered by [Jekyll](https://jekyllrb.com/). Running it on your local machine requires a working [Ruby](https://www.ruby-lang.org/en/) installation with [Bundler](http://bundler.io/).
+Diese Seite wird mit [Jekyll] (https://jekyllrb.com/) betrieben. Für die Ausführung auf Ihrem lokalen Computer ist eine funktionierende [Ruby] (https://www.ruby-lang.org/en/) Installation mit [Bundler] (http://bundler.io/) erforderlich.
```
Comment (jens1o) [inline CONTRIBUTING.md]: spaces yet again

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -2,72 +2,72 @@
...
 
-Once you have that set up, run:
+Sobald Sie das eingerichtet haben, führen Sie Folgendes aus:
 
     script/bootstrap
     script/server
 
-…and open http://localhost:4000 in your web browser.
+…und öffnen Sie http://localhost:4000 in Ihrem Webbrowser.
```
Comment (jens1o) [inline CONTRIBUTING.md]: informal variant

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -2,72 +2,72 @@
...
 
-## Contribution review process
+## Beitragsprüfung
 
-This repo is currently maintained by @nayafia and @bkeepers, who have commit access. They will likely review your contribution. If you haven't heard from anyone in 10 days, feel free to bump the thread or @-mention a maintainer to review your contribution.
+Dieses Repo wird derzeit von @nayafia und @bkeepers verwaltet, die einen Commit-Zugriff haben. Sie werden wahrscheinlich Ihren Beitrag überprüfen. Wenn Sie in 10 Tagen noch nichts von jemandem gehört haben, können Sie dem Thread oder mit einer @ -erwähnung einen Projektleiter hinzufügen, um ihren Beitrag zu überprüfen.
```
Comment (jens1o) [inline CONTRIBUTING.md]: could you replace `nayafia` to `jens1o` please? (You may add you to, if you like to)

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -2,72 +2,72 @@
...
 
-## Community
+## Gemeinschaft
 
-Discussions about the Open Source Guides take place on this repository's [Issues](https://github.com/github/opensource.guide/issues) and [Pull Requests](https://github.com/github/opensource.guide/pulls) sections. Anybody is welcome to join these conversations. There is also a [mailing list](http://eepurl.com/cecpnT) for regular updates.
+Diskussionen über die Open Source Leitfäden finden in den [Issues] dieses Repositories (https://github.com/github/opensource.guide/issues) und [Pull Requests] (https://github.com/github/opensource.guide) statt. Jeder ist willkommen, an diesen Gesprächen teilzunehmen. Es gibt auch eine [Mailingliste] (http://eepurl.com/cecpnT) für regelmäßige Updates.
```
Comment (jens1o) [inline CONTRIBUTING.md]: `Open-Source` and the links

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -2,72 +2,72 @@
...
 
-Wherever possible, do not take these conversations to private channels, including contacting the maintainers directly. Keeping communication public means everybody can benefit and learn from the conversation.
+Wenn möglich, führen Sie diese Gespräche nicht auf privaten Kanälen, einschließlich der direkten Kontaktaufnahme mit den Betreuern. Wenn Sie die Kommunikation öffentlich machen, können alle davon profitieren und von der Konversation lernen.
```
Comment (jens1o) [inline CONTRIBUTING.md]: last: `Du`

File: CONTRIBUTING.md (hunk 9 above)
Comment (jens1o) [reply CONTRIBUTING.md]: @Techassi Do you want to be added here?

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -41,7 +41,7 @@ Bevor wir anfangen, sind hier ein paar Dinge, die wir von dir erwarten (und die
 
 ## Wie man beiträgt
 
-Wenn du etwas beitragen möchtest, suche zunächst in den [Issues] (https://github.com/github/opensource.guide/issues) und [Pull Requests] (https://github.com/github/opensource). .guide / pulls) um zu sehen, ob jemand anderes eine ähnliche Idee oder Frage gestellt hat.
+Wenn du etwas beitragen möchtest, suche zunächst in den [Issues] (https://github.com/github/opensource.guide/issues) und [Pull Requests] (https://github.com/github/opensource.guide/pulls) um zu sehen, ob jemand anderes eine ähnliche Idee oder Frage gestellt hat.
```
Comment (jens1o) [inline CONTRIBUTING.md]: well, could you please update the links too? ;)

File: CONTRIBUTING.md (hunk 2 above)
Comment (Techassi) [reply CONTRIBUTING.md]: Is it fixed?

**Comment by maikksmt** [comment]
This request is redundant due to a recent request

**Pull request closed without merging**